documentation/SDKs/javascript/README.md

```diff
@@ -0,0 +1,358 @@
+#JavaScript SDK for Nether
+
+## Work in Progress
+
+## Downloading the JavaScript SDK for Nether
+
+### Installing the JavaScript SDK for Nether
+
+## Usage
+
+### Analytics
+
+#### Active sessions - daily
+You can use the ***Nether.Analytics.GetActiveSessionsDaily*** method to get the total active sessions per day. Returns an array of ActiveSession
```

Comment:
@stuartleeks Can you confirm that there aren't any specific API's to send analytic events at the moment? Is this in development as all the analytics API's I have are GET methods?